etc/bento.env

```diff
@@ -140,7 +142,7 @@ BENTOV2_SERVICE_REGISTRY_CPUS=1
 
 # Katsu
 KATSU_REPO=https://github.com/bento-platform/katsu
-BENTOV2_KATSU_TAG_OR_BRANCH=v2.14.1
+BENTOV2_KATSU_TAG_OR_BRANCH=v2.14.2
```

Comment:
I'm going to do another tag as `v2.15.0-alpha.1` with release notes and a version bump in setup.py before this gets merged

Reply:
https://github.com/bento-platform/katsu/tree/v2.15.0-alpha.1